src/services/ghost/GhostStreamingParser.ts

```diff
@@ -304,6 +304,21 @@ export class GhostStreamingParser {
 			const searchContent = match[1]
 			// Extract cursor position from replace content
 			const replaceContent = match[2]
+
+			// Validate that extracted content doesn't contain XML tags (indicates regex failure)
```

Comment:
I'm not sure this is a correct fix. I'm pretty sure this cant happen due to the regex, and I havent been able to test this, so I'm not sure adding this is a good idea, as it might just muddle the issue but fix nothing